### PR TITLE
Card comp

### PR DESCRIPTION
**Pull request by harshitpaliwal95**
- Add : card comp
- update links

### CODE REVIEW

File: components/card/card.css [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+  font-weight: 400;
+  font-size: 1.6rem;
+}
+.card-h3 {
+  font-size: 0.8rem;
+  font-weight: 400;
+  color: gray;
+  margin-bottom: 1rem;
+}
+.card-para {
```
Comment (Chiragmodi01) [inline components/card/card.css]: Class could be `.card-description`

File: components/card/card.css [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+  cursor: pointer;
+}
+
+.card-icon-btn-right {
+  margin-left: auto;
+}
+.card-icon {
+  border-radius: 50%;
+  padding: 0.5rem 0.6rem;
+  transition: 0.8s;
```
Comment (Chiragmodi01) [inline components/card/card.css]: Transition time on footer icon & buttons feels very long to me.

File: components/card/card.html [hunk 3]
```diff
@@ -0,0 +1,326 @@
...
+        <span class="main-heading">Light UI</span>
+        <span class="sub-heading">v 1.0</span>
+      </div>
+      <ul class="left-nav">
+        <div class="nav-item"><a href="#">Example</a></div>
+        <div class="nav-item"><a href="#">Github</a></div>
+      </ul>
+    </nav>
+    <!-- ui nav bar -->
+    <div class="main-box">
```
Comment (Chiragmodi01) [inline components/card/card.html:31]: Semantically `main` tag could be used here.

**Comment by harshitpaliwal95** [comment]
@Chiragmodi01 thanks for the feedback ill fix it before merge

**Review by sumitch29 — APPROVED** [review]
Good

File: components/card/card.html [hunk 4]
```diff
@@ -0,0 +1,326 @@
...
+              <div class="card-badge">new</div>
+              <div class="card-comp-contant">
+                <h2 class="card-h2">Happy Coding</h2>
+                <h3 class="card-h3">by Harshit Paliwal</h3>
+                <p class="card-para">
+                  Lorem ipsum dolor sit amet consectetur adipisicing elit. Vel,
+                  tempora.
+                </p>
+              </div>
+              <div class="card-btn-footer">
```
Comment (Chiragmodi01) [inline components/card/card.html:136]: Semantically `footer` tag could be used here.

File: components/card/card.html [hunk 5]
```diff
@@ -0,0 +1,326 @@
...
+                    src="../../assets/code.jpg"
+                    alt="demo"
+                  />
+                </div>
+                <div class="card-comp-contant">
+                  <h2 class="card-h2">Happy Coding</h2>
+                  <h3 class="card-h3">by Harshit Paliwal</h3>
+                </div>
+              </div>
+              <div class="card-btn-footer">
```
Comment (Chiragmodi01) [inline components/card/card.html:246]: Semantically `footer` tag could be used here.

**Review by Chiragmodi01 — COMMENTED** [review]
Can just brush up some classes and use semantic html tags.
Overall looks fine.

**Pull request merged**